pontoon/insights/tasks.py

```diff
@@ -526,6 +575,11 @@ def build_activity_charts_data(start_of_today):
                 data["times_to_review_suggestions"].append(review_time)
                 if action["user"] in pretranslation_users:
                     data["times_to_review_pretranslations"].append(review_time)
+            if action["user"] in pretranslation_users:
+                data["pretranslations_rejected"].add(translation)
+                score = get_chrf_score(action)
+                if score:
+                    data["pretranslations_chrf_scores"].append(score)
```

Comment:
This ends up assigning the pretranslation quality to the date when it's approved or rejected, rather than its creation date. Would it not be more appropriate to use the date when the pretranslation was created?

Reply:
That's an interesting point. OTOH, it would be much more difficult to backdate scores to when the pretranslation was created.

@flodolo Should we add a note to the chart tooltip clarifying that the chrF++ data belongs to translations **reviewed** (not created) in a particular month?

Reply:
To be honest, I'm not sure it's necessary to point it out, since all the data refers to the specific month. With that said, adding it to the tooltip works for me.

Reply:
I'd be ok with a tooltip note for now, as long as there's a code comment also added somewhere around here about the current date solution being used not because we think it's good, but because it's convenient. Then if we do end up refactoring insights due to #2938, it'll be much easier to switch to the pretranslation's date.

Reply:
Is that really the case, though? Maybe the expectation is that the Approval rate line and the chrF++ line are aligned on time? I'll let @flodolo decide what's the actual expectation, because I suspect he'll be the most common use of these charts.

I'll add a code comment and tooltip in any case afterwards (at least the former depends on the expectation).

Reply:
Tooltip updated. I'll add a code comment if the expectation is to calculate the score for the creation month.

Reply:
Here's how I see it:
* Tying a metric to the pretranslation date can help us understand if we make improvements to the system (effect of code changes, retraining models, etc.). 
* Tying a metric to the review date can help us understand (also) what's going on in the locale in a specific period.

Since the quality metric (cHRF++) is an automated score, I think we should tie it to the pretranslation date.

Approval data, on the other hand, is a human score, and it can be influenced by a new person coming in to review for that locale from a specific month, so it makes sense to have it tied to the review date.